mlflow/ml-package-versions.yml

```diff
@@ -148,6 +148,8 @@ lightgbm:
       git clone --recursive https://github.com/microsoft/LightGBM --depth=1 --branch master
       cd LightGBM
       bash build-python.sh install
+      cd ..
+      rm -rf LightGBM
```

Comment:
Instead, we can clone in `/tmp/LightGBM`.

Reply:
Emmm if use /tmp/xxx how do we cd back to this directory?

Reply:
Could we use ./tmp/LightGBM instead?

Reply:
> Emmm if use /tmp/xxx how do we cd back to this directory?

Do we need to?

Reply:
Oh we don't need to cd back, never mind :)

Reply:
If we really need to, we can use pushd and popd.